stock_getter/frontend_stockgetter/scripts/main.js

```diff
@@ -189,4 +198,19 @@ document.addEventListener('DOMContentLoaded', function () {
         });
         console.log(`Data download complete`);
     }
+
+    // Checkbox event listener
+    // Get the pointer to the element
+    let fcstCheckb = document.querySelector('#forecast_check');
+
+    // Add the listener to the checkbox
+    fcstCheckb.addEventListener("change", function() {
+        console.log("Updating status of forecast horizon input");
+        if (this.checked) {

```

Comment:
Can be simplified as:
hrznInput.disabled = !this.checked

Reply:
That's minor. I like curly braces :-)

Reply:
Up to you but kinda bad smell, unnecessary complication. More obvious example for this could be something like:
if (this.checked == true) {
  hrznInput.disabled = true;
} else {
  hrznInput.disabled = false;
}

Reply:
I disagree. You are nitpicking stuff. This is definetely not a code smell, but a syntactic preference. Actually, a lot of coding standards are guiding not to use ternary and use explicit conditional statements for readability. Even though this make take a few more lines of whitespace. Readability is key. So I think we should leave as is.

Reply:
I think you are referring to a different comment about the ternary (I resolved that one), I am ok with that one being a personal preference, I am not being nitpicky about that because based on talking to software engineering people in industry, only about 1 in 5 is against a single ternary. Almost all are against the nested ones, but this is not the case here.

This comment is different. It was a code smell since we were complicating unnecessarily. I saw you already changed the block I was referring to here, so this one seems fine now and I'm resolving it.